pb/summary/summary.proto

```diff
@@ -45,6 +47,74 @@ message FailingTestSummary {
   string fail_test_link = 13;
 }
 
+// Represents the number of tests that have a flakiness at or above the threshold.
+// If a test is counted in this bucket for this threshold, it will not be counted
+// for buckets of a lower threshold.
+//
+// E.g. if a test has a flakiness of 50.0, and there are two buckets: threshold 20.0
+// and threshold 3.0, the test will be counted for the 20.0 bucket and not the 3.0
+// one, even though the flakiness is above both thresholds.
+message FlakyBucket {
+  // Minimum flakiness for the bucket
+  float threshold = 1;
+  // number of tests in the bucket
+  int32 tests = 2;
+}
+
+// Metrics about a specific test, i.e. passes, fails, total runs, etc.
+message TestInfo {
+  // Test name.
+  string name = 1;
+  // Test environment, i.e. staging-1.13
+  string env = 2;
+
+  int32 total_runs = 3;
+  int32 passed_runs = 4;
+  int32 failed_runs = 5;
+  int32 failed_infra_runs = 6;
+  int32 other_runs = 7;
+  float flakiness = 8;
+  int32 bug_id = 9;
+  string infra_issues = 10;
+  int32 total_runs_with_infra = 11;
+}
+
+// Summary of the flakiness and overall healthiness of a dashboard tab
+message HealthinessInfo {
+
+  // The start of the time frame that the analysis was run for.
+  // Represents the lower bound but does no guarantee that the earliest
+  // test occurred at start_date
+  google.protobuf.Timestamp start_date = 1;
+
+  // The end of the time frame that the analysis was run for.
+  // Same caveat as above but for upper bound.
+  google.protobuf.Timestamp end_date = 2;
+
+  // A list of test entries associated with this tab + timeframe.
+  repeated TestInfo tests = 3;
+
+  reserved 4, 5;
+
+  // The flakiness out of 100 (think percentage but drop the sign)
+  float average_flakiness = 6;
```

Comment:
do all flake metrics get *100 to be a percentage?

Reply:
From what I've seen in the old report the initial calculation of this flakiness is the ratio of (failed/total) * 100. The *100 is arbitrary and it is never converted back into a ratio out of 1.0; I don't know for sure if it's to make it a percentage or if *100 is necessary but that makes the most sense to me. I made this comment describing it because it previously never mentioned anything of the sort, and actually I initially thought that flakiness was out of 1.0.